web/docker-compose.yml

```diff
@@ -114,3 +116,5 @@ secrets:
     file: ~/.ssh/id_rsa
   SSH_AUTHORIZED_KEYS:
     file: ~/.ssh/id_rsa.pub
+  SODIUM_CRYPTO_KEY:
+    file: key
```

Comment:
Should this file not be added to .gitignore? Otherwise changes may get checked in with a real key.